src/lighteval/main_accelerate.py

```diff
@@ -75,7 +75,7 @@ def main(args):
     with htrack_block("Model loading"):
         with accelerator.main_process_first() if accelerator is not None else nullcontext():
             model, model_info = load_model(config=model_config, env_config=env_config)
-            evaluation_tracker.general_config_logger.log_model_info(model_info)
+            evaluation_tracker.general_config_logger.log_model_info(model_info if args.model_info is None else args.model_info)
```

Comment:
What is this change for? There should not be a `model_info` in `args` (see `run_evals_accelerate.py` for all args)

Reply:
Oh! I meant to ask you about that. 
The problem is that the model_info doesn't contain any information when using endpoint, so I tried to find a way to inject it. What would be the right way to o do that?

Reply:
I see! Let me think about it!

Reply:
Hm, the model_info is supposed to be created in the load_model function (in model_loader), it should contain the following for an endpoint model
```

    model_info = ModelInfo(
        model_name=model.name,
        model_sha=model.revision,
        model_dtype="default",
        model_size=-1,
    )
    ```

Reply:
Right. So the issue that I encountered was that I had the `model_size` info in the request, but I didn't have any way to insert it into the results, since the results are created and saved entirely in the lighteval library, and there wasn't any way to modify it.